src/Blocks/BlockHandler.cpp

```diff
@@ -1300,7 +1622,7 @@ void Temporary::RegisterAllBlockHandlers(BlockTypeRegistry & aRegistry)
 
 	for (const auto & r: regs)
```

Comment:
No commenting out! If this isn't needed, just delete the whole thing (or at least comment out the for loop as well)!

Reply:
Ah, requesting review for #4855